oss.krtirtho.Spotube.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>oss.krtirtho.spotube</id>
```

Comment:
this is case sensitive and doesn't match with what's in the manifest.

Reply:
Thanks. I'm fixing it in the next commit